generator/document.go

```diff
@@ -283,7 +306,124 @@ func RandomFieldReplace(c chan map[string]interface{}) {
 				"path":  "/Address/City",
 				"value": faker.Word(),
 			}}
-		shuffleAndFillChannel(options, c)
+			shuffleAndFillChannel(options, c)
+		} else if (destination == "elastic") {
+			options := []map[string]interface{}{{
+				"doc": map[string]interface{}{
+					"Email": faker.Email(),
+					"_ts": CurrentTimeMicros(),
+				},
+			},
+			{
+				"doc": map[string]interface{}{
+					"About": faker.Sentence(),
+					"_ts": CurrentTimeMicros(),
+				},
+			},
+			{
+				"doc": map[string]interface{}{
+					"Company": faker.Word() + "-" + faker.Word(),
+					"_ts": CurrentTimeMicros(),
+				},
+			},
+			{
+				"script": map[string]interface{}{
```

Comment:
What's the difference between using script to assign vs. doc? Is it because of nested fields?

Reply:
both are essentially the same https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-update.html. I wasn't able to update nested fields using doc and script worked